.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/hannahschultz/.local/share/virtualenvs/cogitate_tool-HEnOOtpD/bin/python"
+}
```

Comment:
This was being raised in another PR as well: should this file go into the master?

Reply:
I am happy to remove it since it includes only a local path from my device.